velox/functions/prestosql/registration/BinaryFunctionsRegistration.cpp

```diff
@@ -68,6 +68,18 @@ void registerSimpleFunctions(const std::string& prefix) {
       {prefix + "to_ieee754_32"});
   registerFunction<FromIEEE754Bits32, float, Varbinary>(
       {prefix + "from_ieee754_32"});
+  registerFunction<
```

Comment:
I'm seeing that in Presto the signature is (string, size, pad) and here it is (string, pad, size):

```
presto:di> show functions like '%pad';
 Function | Return Type |         Argument Types         | Function Type | Deterministic |          Description          | Variable Arity | Built In | Temporary | Language
----------+-------------+--------------------------------+---------------+---------------+-------------------------------+----------------+----------+-----------+----------
 lpad     | varbinary   | varbinary, bigint, varbinary   | scalar        | true          | pads a varbinary on the left  | false          | true     | false     |
 lpad     | varchar     | varchar(x), bigint, varchar(y) | scalar        | true          | pads a string on the left     | false          | true     | false     |
 rpad     | varbinary   | varbinary, bigint, varbinary   | scalar        | true          | pads a varbinary on the right | false          | true     | false     |
 rpad     | varchar     | varchar(x), bigint, varchar(y) | scalar        | true          | pads a string on the right    | false          | true     | false     |
(4 rows)
```

Reply:
got an earlier comment to change the args to alphabetical order but i changed it just now to match the presto signature (string, size, pad)

Reply:
> got an earlier comment to change the args to alphabetical order 

Maybe we need to let the commenter know that the order of arguments must match Presto. Otherwise, the function won't be resolved.

Reply:
@soumiiow : Masha is correct. Its better the order of arguments match Presto. Apologize I missed that in the review.

Please add a Prestissimo e2e in https://github.com/prestodb/presto/blob/master/presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java for lpad and rpad to ensure the wiring works.